src/api/index.ts

```diff
@@ -40,15 +34,13 @@ const ApiInterventions: { [intervention: string]: string } = {
 type InterventionKey = keyof typeof INTERVENTIONS;
 
 /** Represents timeseries for any kind of region. */
-export type Timeseries = CountyTimeseries | StateTimeseries;
+export type Timeseries = AreaTimeseries;
```

Comment:
Here's an example of awkward typing, should we just `import { Timeseries } from 'schema/CovidActNowAreaTimeseries;'` elsewhere?